Makefile

```diff
@@ -175,6 +175,17 @@ elasticsearch-catalog-build:
 elasticsearch-catalog-deploy:
 	olm_deploy/scripts/catalog-deploy.sh
 
+elasticsearch-catalog-build-local:
+	LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY=127.0.0.1:5000/openshift/elasticsearch-operator-registry make elasticsearch-catalog-build
+
+# deploys the operator registry image with images from quay.io and creates a catalogsource referencing it
+elasticsearch-catalog-deploy-quay:
+	IMAGE_ELASTICSEARCH_OPERATOR=quay.io/openshift/origin-elasticsearch-operator:latest \
```

Comment:
do you need this?  Does jaeger CI not have access to the openshift CI registry?

Reply:
The whole PR is a result of me pairing today with @pavolloffay on this topic. Here some new findings:
- The Jaeger team is not part of the openshift GH organization
- The Jaeger team has no access to our registry.ci.openshit.org

Reply:
@jcantrill many of the targets in this repository depends on internal stuff - e.g. access to openshift CI. This makes it impossible to use the deploy and other targets from this repository by any OSS community member. 

It would be much easier to work with this operator if the assumption was that anybody who as access to github can build and deploy it. For your team there can be a swich that will use internal resources.